### PR TITLE
Show current playlist

### PR DESCRIPTION
**Pull request by VuManhHung97**
Create playSonglistener to communicate between activity and fragment
Transfer Service Connection to activity
Handle send song list via PlaySong screen
Show current playlist
Handle when click current playlist

https://edu-redmine.sun-asterisk.vn/issues/12958

current_playlist

### CODE REVIEW

File: app/src/main/java/com/sun_asterisk/music_65/screen/controlsong/ControlSongFragment.java [hunk 1]
```diff
@@ -166,8 +111,7 @@ public void onMediaStateChange(boolean isPlaying) {
         }
     }
 
-    @Override
-    public void onLoop(int loopType) {
+    public void onSetLoop(int loopType) {
```
Comment (thaohq-0814) [inline app/src/main/java/com/sun_asterisk/music_65/screen/controlsong/ControlSongFragment.java]:
```suggestion
    public void onLoopChanged(int loopType) {
```

File: app/src/main/java/com/sun_asterisk/music_65/screen/controlsong/ControlSongFragment.java [hunk 2]
```diff
@@ -148,15 +101,7 @@ public void onClick(View v) {
...
-    @Override
-    public void onSongChange(Song song) {
-        if (song != null) {
-            updateUI(song);
-        }
-    }
-
-    @Override
-    public void onMediaStateChange(boolean isPlaying) {
+    public void updatePlay(boolean isPlaying) {
```
Comment (thaohq-0814) [inline app/src/main/java/com/sun_asterisk/music_65/screen/controlsong/ControlSongFragment.java]:
```suggestion
    public void onPlaySateChanged(boolean isPlaying) {
```

File: app/src/main/java/com/sun_asterisk/music_65/screen/controlsong/ControlSongFragment.java [hunk 3]
```diff
@@ -181,8 +125,7 @@ public void onLoop(int loopType) {
         }
     }
 
-    @Override
-    public void onShuffle(boolean isShuffle) {
+    public void onSetShuffle(boolean isShuffle) {
```
Comment (thaohq-0814) [inline app/src/main/java/com/sun_asterisk/music_65/screen/controlsong/ControlSongFragment.java]:
```suggestion
    public void onShuffleStateChanged(boolean isShuffle) {
```

File: app/src/main/res/layout/item_playlist.xml [hunk 4]
```diff
@@ -3,24 +3,46 @@
...
+        android:id="@+id/textTitleSong"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginLeft="@dimen/dp_24"
-        android:maxWidth="@dimen/dp_200"
-        android:textSize="@dimen/sp_20"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"/>
-
+        android:layout_marginEnd="8dp"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/item_playlist.xml]: hardcode

**Comment by VuManhHung97** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**